pkg/options/options.go

```diff
@@ -16,11 +16,13 @@ limitations under the License.
 
 package options
 
-// PowerVSProviderIDFormat is used to identify the Provider ID format for Machine.
-var PowerVSProviderIDFormat string
+const (
+	// PowerVSProviderIDFormatV1 will set provider id to machine as ibmpowervs://<cluster_name>/<vm_hostname>
+	PowerVSProviderIDFormatV1 = "v1"
 
-// PowerVSProviderIDFormatV2 will set provider id to machine as ibmpowervs://<region>/<zone>/<service_instance_id>/<powervs_machine_id>
-const PowerVSProviderIDFormatV2 = "v2"
+	// PowerVSProviderIDFormatV2 will set provider id to machine as ibmpowervs://<region>/<zone>/<service_instance_id>/<powervs_machine_id>
+	PowerVSProviderIDFormatV2 = "v2"
```

Comment:
I guess these 2 can be of type `PowerVSProviderIDFormat` right?

Reply:
Yes, Correct I will update

Reply:
But if i do so , It needs additional conversions from `PowerVSProviderIDFormat` to `string` during validating flags in main.go file.